test/simple.ml

```diff
@@ -1,5 +1,6 @@
 (* We hide the context by instanciating a functor: *)
-module Z = ZZ3.Make (struct let ctx = Z3.mk_context [] end)
+module Z = Z3overlay.Make (struct let ctx = Z3.mk_context [] end)
+(* module Z = ZZ3.Make (struct let ctx = Z3.mk_context [] end) *)
```

Comment:
Get rid of the old one, it's deprecated !

Reply:
Indeed.